tests/Backend/ExternalBuckets/TeradataRegisterBucketTest.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Keboola\Test\Backend\ExternalBuckets;
+
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\Test\Backend\Workspaces\Backend\TeradataWorkspaceBackend;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Utils\EventsQueryBuilder;
+
+class TeradataRegisterBucketTest extends BaseExternalBuckets
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->initEvents($this->_client);
```

Comment:
toto volas v kazdom teste na pozadovanom mieste tu to imho teda nemusi byt, zbytocne sa 2x zaklada dummy eventa